tests/svg.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>SVG</title>
+  <script src="../../web-component-tester/browser.js"></script>
+  <script>
+    window.ShadyDOM = {force: true};
+    window.ShadyCSS = {shimshadow: true};
+  </script>
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+</head>
+<body>
+  <template id="svg-in-shadow">
+    <style>
+      :host {
+        display: inline-block;
+        background-color: #ccc;
+      }
+
+      .test-class {
+        border: 3px solid blue;
+      }
+
+      circle {
+        fill: blue;
+      }
+    </style>
+    <svg
+      xmlns="http://www.w3.org/2000/svg" version="1.1"
+      width="100px" height="100px" viewBox="0 0 100 100"
+      class="test-class"
+    ></svg>
+  </template>
+  <script>
+    window.addEventListener('WebComponentsReady', function() {
```

Comment:
This can all live in the top level suite inside of a `suiteSetup` function, and you won't need the event listener for WebComponentsReady.